### ISSUE TITLE
Better acessibility support

### ISSUE THREAD

**Issue by Gwenwyfar** [SimpleMachines/SMF#4595]
I was reading more about web accessibility support and we are lacking in a lot of things such as proper label/tags use, as well as use of role tags, etc. It calls for a revision on the templates to make sure we have at least all the basics. I don't think we need this for RC, but this should be done before 2.1 goes gold.

Once I've researched this better I'll make a list of everything to review on SMF.

(And yep, Antechinus, you convinced me ;) )

**Comment by Sesquipedalian** [comment]
Excellent! I am strongly in favour of this. Please do, @Gwenwyfar!

**Comment by StealthWombat** [comment]
Just a FYI: I don't know if you've been keeping an eye on what the wild elks are up to, but I noticed these today: https://github.com/elkarte/Elkarte/issues/1981 and https://github.com/elkarte/Elkarte/issues/2994

Since Vekseid has actual people testing this sort of stuff, I'm going to go through his reported issues and PR's to see if I can spot anything relevant for SMF.

**Comment by sbulen** [comment]
Although I agree with the intention, this request is far too broad and in its current state is inactionable.

SMF does adhere to most accessibility requirements. But not all. 

To fix this, we need specifics.  This PR does not provide specifics.  

@MissAllSunday I think **_we should at least remove the RC4 label..._**

Maybe even close the issue.  If this is serious for someone, we need specifics...  There is nothing actionable here.

**Comment by m4z** [comment]
A more concrete recent discussion with some useful links: [WCAG Accessibility Standards](https://www.simplemachines.org/community/index.php?topic=577088)

**Comment by MissAllSunday** [comment]
Yes I will be in favor of incrementing accesibility support, however we need to define how we will tackle this issue and that needs time and proper preparation, therefore this will be a nice feature to include on a future 2.x release.
